app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux'
+import { getRandomGreeting } from '../Redux/Greetings/greetingsReducer'
+
+const Greeting = () => { 
+  const dispatch = useDispatch();
+  const { greeting, error } = useSelector((state) => state.greetings);
+
+  useEffect(() => {
+    dispatch(getRandomGreeting());
+  }, [dispatch]);
+  console.log(greeting)
+
+  
```

Comment:
- [ ] For **[JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)**, kindly make sure you do not commit `console.log` to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.